docs/specs/layers.md

```diff
@@ -18,6 +18,30 @@ The `ty` property defines the specific layer type based on the following values:
 
 {schema_object:layers/visual-layer}
 
+
+#### Mattes
+
+A matte allows using a layer as a mask for another layer.
```

Comment:
Can any layer type be used as a mask?

Reply:
All the layer types defined so far (AFAIK)